src/collective/solr/commands.py

```diff
@@ -11,9 +14,15 @@
 
 
 def _get_site(app, args):
-    name = None
-    if len(args) > 0:
-        name = args[0]
+    # Zope.Startup.zopectl.ZopeCmd.run_entrypoint promises to pass the entry point's
+    # name as the first argument and any further arguments after that, but that does not 
+    # work with plone.recipe.zope2instance. Using positional arguments therefore
+    # is unreliable - resolve to using a flag.
+    parser = argparse.ArgumentParser()
```

Comment:
@tschorr if we rely on argparse, don't we have to add it to the dependencies (Python 2.7)?